packages/account/src/utils/index.ts

```diff
@@ -1,3 +1,3 @@
-export * from "./Types";
-export * from "./Utils";
-export * from "./Constants";
+export * from "./Types.js";
+export * from "./Utils.js";
+export * from "./Constants.js";
```

Comment:
Why is this needed

Reply:
For esm builds. https://stackoverflow.com/questions/62619058/appending-js-extension-on-relative-import-statements-during-typescript-compilat/62626938#62626938

Reply:
In the new SDK we have a more robust solution: tsc-alias
https://github.com/bcnmy/sdk/blob/main/package.json#L25